src/test/crimson/seastore/test_object_data_handler.cc

```diff
@@ -297,7 +297,7 @@ struct object_data_handler_test_t:
       "seastore_max_data_allocation_size", "8192").get();
   }
 
-  laddr_t get_random_laddr(size_t block_size, laddr_t limit) {
+  size_t get_random_laddr(size_t block_size, size_t limit) {
```

Comment:
size_t -> objaddr_t

Reply:
How about renaming it to `get_random_write_offset` ? The return value is not a laddr_t.